docs/en/docs/tutorial/dependencies/classes-as-dependencies.md

```diff
@@ -101,7 +101,7 @@ In both cases the data will be converted, validated, documented on the OpenAPI s
 
 Now you can declare your dependency using this class.
 
-And as when **FastAPI** calls that class the value that will be passed as `commons` to your function will be an "instance" of the class, you can declare that parameter `commons` to be of type of the class, `CommonQueryParams`.
+**FastAPI** calls that class. This creates an "instance" of that class. The instance will be passed as `commons` to your function. You can declare that parameter `commons` to be of type of the class, `CommonQueryParams`.
```

Comment:
```suggestion
**FastAPI** calls that class. This creates an "instance" of that class. Then the instance will be passed as `commons` to your function and you can declare that parameter `commons` to be of type of the class, `CommonQueryParams`.
```

Reply:
How about

'Then the instance will be passed as parameter `commons` to your function and you can declare this parameter to be of type of the class, `CommonQueryParams`.

To be honest I dont know how to make this change, i just see a 'Commit suggestion' button, but no 'Edit' button. I fear breaking something and Git doesnt really let me undo stupid things :-)

Reply:
You'll need to change locally or you can create your own suggestion and apply hahaha

Reply:
But I agree with your changes :)

Reply:
Not sure what you mean with missing part (probably because its missing ;-) )

Reply:
Ah, ok, thanks

Reply:
Reformulated this part a bit. Let me know if you like it.